.github/workflows/depsreview.yml

```diff
@@ -4,6 +4,7 @@ on: [pull_request]
 jobs:
   dependency-review:
     runs-on: ubuntu-latest
+    if: github.repository_owner == 'envoyproxy'
```

Comment:
great, i think we also want the repo name (not sure what the var is) to be `'envoy'`